D1673/P1673.md

```diff
@@ -6496,29 +6519,34 @@ reference.  It is part of the implementation of `scaled`
 ```c++
 template<class ScalingFactor,
          class Accessor>
-class accessor_scaled {
+class scaled_accessor {
 public:
-  using reference = scaled_scalar<ScalingFactor,
-    typename Accessor::reference,
-    typename Accessor::element_type>;
   using element_type = add_const_t<typename reference::value_type>;
```

Comment:
Looks like we need to update this line too.  (If `reference` is `element_type`, then `reference::value_type` likely doesn't exist or mean what we want.)

```suggestion
  using element_type = decltype(declval<ScalingFactor>() * declval<Accessor::element_type>());
```

Reply:
we need to talk about this a bit more.